widgets/common/src/RevertToDefaultIcon.jsx

```diff
@@ -39,6 +50,8 @@ RevertToDefaultIcon.defaultProps = {
     value: undefined, // value can be undefined
     defaultValue: undefined, // defaultValue can be undefined
     onClick: _.noop,
+    onMouseDown: _.noop,
```

Comment:
One thing I want to bring up generally is that I often see noops assigned as default functions. I agree with it when the function is actually required, but, to me, it does not make sense when the function is optional, such as here :slightly_smiling_face: 

If `Icon` can happily accept `undefined` instead of a valid function (I have not checked that in the code, but I am pretty sure it will), then we can save ourselves attaching event listeners (even if they are quite cheap) and the small overhead of calling them.

Thus, I would encourage using `undefined` as a default value for optional callbacks, rather than a valid function that does nothing :slightly_smiling_face: 

Assigning default noop functions was what caused a bug to appear in ui-components that was later fixed by https://github.com/cloudify-cosmo/cloudify-ui-components/pull/59

```suggestion
    onClick: undefined,
    onMouseDown: undefined,
```

Reply:
Yes, I agree. Changed.